File: backend/GestionDeUsuarios/GestionDeUsuarios/Commons/RequestResult.cs
namespace GestionDeUsuarios.Commons
{
    public class RequestResult
    {
        public bool Success { get; set; }
        public string Message { get; set; }
        public object Result { get; set; }

        public static RequestResult Unauthorized = new RequestResult
        {
            Message = "Unauthorized"
        };
    }
}

File: frontend/src/app/endpoints/requestResult.ts
export class requestResult {

    public success: boolean = false;
    public message:string = "";
    public result:any;
    public errors:object;

    constructor() {
        this.success = false;
        this.message = "Not data available";
        this.result = {};
        this.errors = {};
    }
    
}